### PR TITLE
[NUI] Clean up SubWindowTest.cs

### PR DESCRIPTION
**Pull request by dongsug-song**
### Description of Change ###
[NUI] Clean up SubWindowTest.cs
- change testcase methods' names.
- change case's variable as constant.

### API Changes ###
none

### CODE REVIEW

File: test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/SubWindowTest.cs [hunk 1]
```diff
@@ -109,144 +149,142 @@ public void WinTouchEvent(object sender, Window.TouchEventArgs e)
...
-            tm = new Timer(3000);
-            tm.Tick += Tm_Tick;
-            tm.Start();
+            timer = new Timer(3 * oneSecond);
+            timer.Tick += TimerTick;
+            timer.Start();
         }
 
-        private bool Tm_Tick(object source, Timer.TickEventArgs e)
+        private bool TimerTick(object source, Timer.TickEventArgs e)
```
Comment (everLEEst) [inline test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/SubWindowTest.cs:262]: I suggest name DisposeTimer or DisposeSubWinSecondTimer to know what this timer should be supposed to do.

Comment (dongsug-song) [reply test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/SubWindowTest.cs]: Thank you for the review.
I fixed it.

**Review by Seoyeon2Kim — APPROVED** [review]
LGTM. This PR is to update code quality of a sample.

**Review by jaehyun0cho — APPROVED** [review]
LGTM

**Review by everLEEst — APPROVED** [review]
Looks good to me:)

**Pull request merged**